PendingReleaseNotes

```diff
@@ -12,6 +12,9 @@
   of the column showing the state of a group snapshot in the unformatted CLI
   output is changed from 'STATUS' to 'STATE'. The state of a group snapshot
   that was shown as 'ok' is now shown as 'complete', which is more descriptive.
+* CephFS: `fallocate` now returns `EOPNOTSUPP` (Operation not supported) for
+  the default case since CephFS does not support disk space reservation. The
```

Comment:
nit: just mention that the default case is mode:0

Reply:
updated the `PendingReleaseNotes` with `mode == 0`